README.md

```diff
@@ -10,6 +10,8 @@ More installation information and options: <https://docs.brew.sh/Installation>.
 
 If running Linux or WSL, [there are some pre-requisite packages to install](https://docs.brew.sh/Homebrew-on-Linux#requirements).
 
+You can set `HOMEBREW_NO_INSTALL_FROM_API` to tap Homebrew/homebrew-core otherwise, by default, it will not be tapped as it is no longer necessary.
```

Comment:
```suggestion
You can set `HOMEBREW_NO_INSTALL_FROM_API` to tap Homebrew/homebrew-core; by default, it will not be tapped as it is no longer necessary.
```